packages/@okta/vuepress-site/guides/sign-into-spa/section12/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Next Steps
+---
+
+Nice job! You should understand how to sign users into your single-page applications using Okta. If you got stuck, post a question in our [Developer Forums](https://devforum.okta.com).
```

Comment:
Nice job! You should understand how to sign users in to your single-page applications using Okta. If you got stuck, post a question in our [Developer Forums](https://devforum.okta.com).